### PR TITLE
WordPress Guide

### PR DESCRIPTION
**Pull request by digamber89**
## Change Summary
Added guide for WordPress Plugin

## PR Checklist
- [x] I have read and signed the [Contributor License Agreement](https://forms.gle/PZyiY5N2GDQU8GsV9).

### CODE REVIEW

File: docs-site/content/guide/wordpress-integration.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+If for some reason automatic installation is not possible, go to [https://wordpress.org/plugins/search-with-typesense/](https://wordpress.org/plugins/search-with-typesense/) , and you will see the download button. Clicking download button will provide you with a zip file of the plugin then.
+- Go to WordPress Admin area > Plugins > Add New and click upload plugin.
+- Click upload plugin and then add the zip file
+- The plugin will then be installed, then activate the plugin.
+
+Once the plugin has been installed and activated - we need to do someone initial setup.
+
+## Setup and Configuration
+
+### Setup
```
Comment (jasonbosco) [inline docs-site/content/guide/wordpress-integration.md:30]: Could you include a screenshot of the WP Admin configuration section for the plugin, with some sample values filled out?

Here's an example of how to include images: https://raw.githubusercontent.com/typesense/typesense-website/master/docs-site/content/guide/dynamodb-full-text-search.md (Search for "images" in that page).

File: docs-site/content/guide/wordpress-integration.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+The fields are
+- Protocol
+- Node URI
+- Port
+- Admin API Key
+- Search API Key
+
+Your WordPress install will now be able to connect to your typensese instance.
+
+### Configuration
```
Comment (jasonbosco) [inline docs-site/content/guide/wordpress-integration.md:51]: Could you include a screenshot for this as well?

File: docs-site/content/guide/wordpress-integration.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+#### Enable Post Types
+This option allows you to select which post types to index. By default the plugin allows you to index posts and pages. There are WordPress action hooks and filters that allow adding custom post types.
+
+#### Bulk Index
+Bulk Index, option allows you to index all posts(post types) at once. Indexing means they will add the document into Typesense instant to be easily retrieved. There is option to 
+- Index
+- Delete and Re-index
+Delete and Re-index is only required if you have modified the schema (advanced usage)
+
+#### Replace WordPress Search
```
Comment (jasonbosco) [inline docs-site/content/guide/wordpress-integration.md:70]: Could you include a screenshot showing the search box?

**Review by jasonbosco — CHANGES_REQUESTED** [review]

File: docs-site/content/guide/wordpress-integration.md (hunk 1 above)
Comment (digamber89) [reply docs-site/content/guide/wordpress-integration.md:30]: Added images / screenshots for overall guide.

File: docs-site/content/guide/wordpress-integration.md (hunk 2 above)
Comment (digamber89) [reply docs-site/content/guide/wordpress-integration.md:59]: Added images / screenshots for overall guide.

File: docs-site/content/guide/wordpress-integration.md (hunk 3 above)
Comment (digamber89) [reply docs-site/content/guide/wordpress-integration.md:80]: Added images / screenshots for overall guide.

**Review by jasonbosco — APPROVED** [review]

**Pull request merged**